src/api/auth/authHandler.ts

```diff
@@ -0,0 +1,169 @@
+import express from 'express';
+import EmailService, { VERIFY_TEMPLATE, RESET_TEMPLATE } from '../../util/emailService';
+import getDatabase from '../../database';
+import * as constants from '../../common/constants';
+
+const EmailToken = getDatabase().getEmailToken();
+const Users = getDatabase().getUsers();
+const randomstring = require('randomstring');
+const sha256 = require('sha256');
+
+
+export class PassportUtil {
+  static getPassportSession = (req: express.Request) => {
+    const result = (req.session && req.session.passport) ? req.session.passport.user : undefined;
+    return result;
+  };
+}
```

Comment:
굳이 클래스일 필요가 없는 단순 유틸 함수입니다.

단순하게 const 문 하나로 처리할 수 있을 것 같습니다.

```suggestion
const getPassportSession = (req: express.Request) => (req.session && req.session.passport) ? req.session.passport.user : undefined;
```

Reply:
유닛 테스트를 위해서 sinon 라이브러리를 사용하고 있고, 해당 라이브러리는 특정 object에 있는 매소드를 트리거하여, 조작을 할 수 있습니다. 그래서 passport session 검사 로직을 유닛 테스트할 때, object 형태로 컨트롤하려고, class를 생성했습니다.

Reply:
만약 저 함수를 이 파일에서만 쓰는 것이라면, AuthHandler 자체를 static class로 만들고 모든 핸들러를 static으로 그 안에 넣은 뒤에 저 함수를 유틸 함수의 성격으로 편입시킨 뒤 sinon에서 직접 그 Object를 불러다 내부 함수를 호출할 수 있도록 설계하면 더 깔끔할 것 같습니다.